hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/InternalConstants.java

```diff
@@ -182,4 +182,9 @@ private InternalConstants() {
       Collections.unmodifiableSet(
           new HashSet<>(Arrays.asList(Constants.FS_S3A_CREATE_PERFORMANCE)));
 
+  /**
+   *  The default scheme for S3A filesystem.
+   */
+  public static final String S3A_SCHEME_DEFAULT = "s3a";
```

Comment:
Instead of defining a new constant, can you reuse `org.apache.hadoop.fs.s3a.Constans#FS_S3A`?